02 - Community Expansions/02.01 Plugins by Category/Task management plugins.md

```diff
@@ -24,15 +24,18 @@ publish: true
 - [[todo-txt|Todo.txt support]]
 - [[obsidian-kanban|Kanban]]
 - [[obsidian-tasks-plugin|Tasks]]
-- [[obsidian-pomodoro-plugin|Obsidian Pomodoro Plugin]]
 - [[tq-obsidian|tq]]
 - [[obsidian-statusbar-pomo|Status Bar Pomodoro Timer]]
 - [[obsidian-random-todo|Random To-Do]]
 - [[obsidian-reminder-plugin|Reminder]]
 - [[imdone-obsidian-plugin|Open cards in imdone from obsidian.]]
 - [[obsidian-amazingmarvin-plugin|Amazing Marvin]]
-- [[obsidian-toggl-integration|Toggl Track Integration]]
 - [[obsidian-task-archiver|Obsidian Task Archiver]]
+- [[obsidian-apple-reminders-plugin|Apple Reminders]]
+- [[things-logbook|Things Logbook]]
+
```

Comment:
I can review all the lists after this change, to find any split lists...

Reply:
I think there shouldn't be any more split lists, let me know if you find any additional ones (last commit should've fixed this one)